kaldb/src/main/java/com/slack/kaldb/logstore/opensearch/OpenSearchAggregationAdapter.java

```diff
@@ -542,7 +710,11 @@ protected static TermsAggregationBuilder getTermsAggregationBuilder(TermsAggBuil
     }
 
     for (AggBuilder subAggregation : builder.getSubAggregations()) {
-      termsAggregationBuilder.subAggregation(getAggregationBuilder(subAggregation));
+      if (isPipelineAggregation(subAggregation)) {
```

Comment:
NBD - but if the pipeline aggregation hasn't been implemented yet, we'll throw an IllegalArgumentException from the else block saying `Aggregation type %s not yet supported"`

I think that's okay for now? Just wanted to highlight it though

Reply:
This is correct - we have the same logic in place for the non-pipeline aggregators. This is to prevent issues attempting to query aggregations that aren't supported, and getting unexpected results. 

When querying from Grafana via envoy this manifests as an http 500.